assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -38,6 +38,18 @@ export const oldContributionsFlowTests: Tests = {
 // Participations in these tests are only assigned
 // to browsers landing on pages/new-contributions-landing/contributionsLanding.jsx
 export const newContributionsFlowTests: Tests = {
+  annualContributionsRoundThree: {
+    variants: ['control', 'annualAmountsA'],
```

Comment:
how come the variants here are 'control' and 'annaulAmountsA' yet above it's 'annualAmountsA' and 'annualAmountsB'?

Reply:
ooh, good spot. Happened during the rebase when the tests were broken out into old/new